src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -726,6 +726,45 @@ public void BuildApplicationWithSpacesInPath ([Values (true, false)] bool enable
 				var dexFile = b.Output.GetIntermediaryPath (Path.Combine ("android", "bin", "classes.dex"));
 				FileAssert.Exists (dexFile);
 				Assert.IsTrue (DexUtils.ContainsClass (className, dexFile, AndroidSdkPath), $"`{dexFile}` should include `{className}`!");
+				var classes = new [] {
+					"Lmono/MonoRuntimeProvider;",
+					"Landroid/runtime/JavaProxyThrowable;",
+					$"L{toolbar_class.Replace ('.', '/')};"
```

Comment:
This line is causing a compilation failure: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=5344444&view=logs&jobId=96fd57f5-f69e-53c7-3d47-f67e6cf9b93e&j=96fd57f5-f69e-53c7-3d47-f67e6cf9b93e&t=65256bb7-a34c-5353-bc4d-c02ee25dc133

```
…/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs(732,10): error CS0103: The name 'toolbar_class' does not exist in the current context
…/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs(734,18): error CS0136: A local or parameter named 'className' cannot be declared in this scope because that name is used in an enclosing local scope to define a local or parameter
```